src/commands/protocols/common/epoch-check/blockchain-epoch-check-command.js

```diff
@@ -54,7 +54,7 @@ class BlockchainEpochCheckCommand extends Command {
         // We don't expect to have this many transactions in one epoch check window.
         // This is just to make sure we don't schedule too many commands and block the queue
         // TODO: find general solution for all commands scheduling blockchain transactions
-        totalTransactions = Math.min(totalTransactions, COMMAND_QUEUE_PARALLELISM * 0.3);
+        totalTransactions = Math.min(totalTransactions, Math.floor(COMMAND_QUEUE_PARALLELISM / 3));
```

Comment:
Why did you change this part?

Reply:
We added *0.5 as we added gnosis chain, and changed it to 0.3 when base was added, not to let single chain. With 0.3 we let 10%  of queu go unused so just fixed that.

When I thing about maybe it makes sense to go with / number_of_enabled_chains